misc/factorial.c

```diff
@@ -1,31 +1,24 @@
-#include <stdio.h>
+#include<stdio.h>
+
 int main()
 {
-    int a[200], n, counter, temp, i;
-    a[0] = 1;
-    counter = 0;
-    printf("Enter a whole number to Find its Factorial: ");
-    scanf("%d", &n);
-    if (n < 0)
-        printf("Cannot Calculate factorials for negative numbers.");
-    else
-    {
-        for (; n >= 2; n--)
-        {
-            temp = 0;
-            for (i = 0; i <= counter; i++)
-            {
-                temp = (a[i] * n) + temp;
-                a[i] = temp % 10;
-                temp = temp / 10;
-            }
-            while (temp > 0)
-            {
-                a[++counter] = temp % 10;
-                temp = temp / 10;
-            }
-        }
-        for (i = counter; i >= 0; i--) printf("%d", a[i]);
-    }
-    return 0;
+	long long int n, factorial = 1;
```

Comment:
Here you have conflicting type error. You have declared the variable ``n`` already previously. Please fix this issue.

Reply:
Where is the "other" `n` defined? I only see 1 variable named `n`.

Reply:
@Panquesito7, I made a mistake! My bad. I think the code that I compiled includes the code which was deleted as well. Lemme delete my review. Thanks for pointing out.